Sources/FluentDarkModeKit/Extensions/UILabel+DarkModeKit.swift

```diff
@@ -19,14 +29,6 @@ extension UILabel {
     set { objc_setAssociatedObject(self, &Constants.currentThemeKey, newValue, .OBJC_ASSOCIATION_COPY_NONATOMIC) }
   }
 
-  @objc private dynamic func dm_didMoveToWindow() {
-    dm_didMoveToWindow()
-    if currentUserInterfaceStyle != DMTraitCollection.current.userInterfaceStyle {
-      currentUserInterfaceStyle = DMTraitCollection.current.userInterfaceStyle
-      dmTraitCollectionDidChange(nil)
-    }
-  }
-
   override open func dmTraitCollectionDidChange(_ previousTraitCollection: DMTraitCollection?) {
```

Comment:
is this redundant?

Reply:
looks like it is

Reply:
We should call `updateDynamicColorInAttributedText` in iOS 11, it seems we accidentally dropped the call when we drops support for iOS 11.

Reply:
@kylinroc  that makes sense. I'll add it back since we support iOS 11 in this library. is it before or after calling super? And it might be a good idea if we add a comment demonstrating the reason why it is needed on iOS 11.

Reply:
We should add it back only for iOS 11 then.

Reply:
Fixed